pkg/test/detector/maven.go

```diff
@@ -0,0 +1,61 @@
+package detector
+
+import (
+	"strings"
+
+	apiClient "github.com/kubeshop/testkube/pkg/api/v1/client"
+	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
+)
+
+// has multiple files and a pom.xml file
+
+// MavenAdapter is detector adapter for Maven test
+type MavenAdapter struct{}
```

Comment:
Sounds like,we have multiple copies of this behaivot.Might be to add DefaultAdapter with a few params?

Reply:
This is what I am working on next